src/contracts/implementations/arbitrable/MultipleArbitrable.js

```diff
@@ -55,8 +35,8 @@ class MultipleArbitrable extends ContractImplementation {
       0,
       'latest',
       {
-        _transactionId: this.arbitrableTransactionId,
-        _evidence: this.metaEvidenceJsonLink
+        _transactionId: arbitrableTransactionId,
+        _evidence: metaEvidenceJsonLink
```

Comment:
Won't work. Not indexed

Reply:
Also it looks like MetaEvidence is not following the standard in `MultupleArbitrableTransaction` (it calls `_metaEvidenceID` -> `_transactionId` in the event). This makes it so it is incompatible with the api. Can you change it to follow the standard? It needs to change in the `Dispute` event as well